### PR TITLE
Semihalf updates

### PR DESCRIPTION
**Pull request by semihalf-wojtas-marcin**
Hi,

Please check Semihalf updates for 2019q1 period.

Best regards,
@mw

### CODE REVIEW

File: 2019q1/bcm58712.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+## Broadcom ARM64 SoC support ##
+
+Contact: Michal Stanek, <mst@semihalf.com>, Marcin Wojtas, <mw@semihalf.com>
```
Comment (emaste) [inline 2019q1/bcm58712.md]: Probably don't want comma between name and <> address. Could be fixed in post-merge edit but if you resubmit for other reasons please include it too.

File: 2019q1/bcm58712.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+Semihalf team work continuation on a FreeBSD support for Broadcom BCM5871X SoC series.
```
Comment (emaste) [inline 2019q1/bcm58712.md]: Minor edit:

The Semihalf team continued working on FreeBSD support for the Broadcom BCM5871X SoC series.

You can make it an inline link if you like
[Broadcom BCM5871X SoC series](https://...)

File: 2019q1/bcm58712.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+   * OTP (One Time Programmable memory) driver
+
+* In progress:
+  * BNXT Ethernet support
+  * Crypto engine acceleration for IPsec offloading.
+
+* Todo:
+  * Upstreaming - this work is expected to be submitted to HEAD in the second half of 2019.
+
+Sponsored by: Juniper
```
Comment (emaste) [inline 2019q1/bcm58712.md]: Should that be `Juniper Networks` or `Juniper Networks, Inc.` perhaps?

File: 2019q1/ena.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+Since the last update internal review and improvements of the patches were
+done, followed by validation on various AWS instances.
+
+To do:
+
+* Upstream of the ENAv2 patches
+
+Recently, AWS released the A1 instances which are arm64 instances. Kernel was
+fixed, so the ENA can be used on those instances without any issues. Changes
+required adjustments in resource activation in the ENA driver (r345371) and
```
Comment (emaste) [inline 2019q1/ena.md]: Make these revs inline links perhaps?

Comment (semihalf-wojtas-marcin) [reply 2019q1/ena.md]: Done

File: 2019q1/bcm58712.md (hunk 3 above)
Comment (semihalf-wojtas-marcin) [reply 2019q1/bcm58712.md]: Done

File: 2019q1/bcm58712.md (hunk 2 above)
Comment (semihalf-wojtas-marcin) [reply 2019q1/bcm58712.md]: Done x2

File: 2019q1/bcm58712.md (hunk 1 above)
Comment (semihalf-wojtas-marcin) [reply 2019q1/bcm58712.md]: Done

**Pull request merged**